### DISCUSSION TITLE
Displaying problem, too many tags

### DISCUSSION THREAD

**Discussion by cyfer-luis** [osmandapp/OsmAnd#20169] [Q&A]
Good morning,
I've problem with the tags displaying on osmand, it's showing too many informations and it's very confusing and unusefull.
The device is a 
realme gt master
android 13
Resolution 1080x2400. 
<table style="width:100%">
  <tr>
<th><img src="https://github.com/osmandapp/OsmAnd/assets/14943610/57f78227-adeb-444e-8334-8625eb131ff4"  width="300" height="auto"/> </td>
  </tr>
</table>

Thanks
Adriano

**Comment by mikehgentry** [comment]
Looks like you've inadvertently adjusted the map magnifier. Try long-pressing on the zoom buttons

https://osmand.net/docs/user/map/vector-maps/#map-magnifier

**Reply by cyfer-luis** [reply]
Map magnifier and text size setting to 100%
I've changed some setting, with same results.
